base-pom/pom.xml

```diff
@@ -567,6 +555,11 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+        <id>jrs-rest-java-client</id>
+        <name>Jaspersoft clients releases</name>
+        <url>http://jaspersoft.artifactoryonline.com/jaspersoft/jaspersoft-clients-releases/</url>
+    </repository>
```

Comment:
why is the jasper repo included here? isn't it only needed for the reporting project?
also this will lead to multiple sources for jasper since we've added 
http://maven.synventis.com/externals/com/jaspersoft/jrs-rest-java-client/
Could this lead to potential issues?

Reply:
repositories are all mentioned in the base-pom not in project poms.
Why do we mirror the jaspersoft repository?

Reply:
the most current release we required wasn't pushed to their repository. also it looks sloppy in general (e.g. no sources.jar, jar hashes, etc.). hence we decided to publish on our external in orderly fashion.
I suggest we don't include it here if there is no deeper reason to it.

Reply:
ok, I will double-check my maven-settings. Maybe the synventis-external repository is missing in my user-settings. Should we add the synventis-external repository to the base-pom?

Reply:
yes you need to update your maven settings (see my slack message).
synventis-external is closed source. do we have a celements-external we could include in the base-pom?

Reply:
yes, line 509 'celements-externals' can we move the jaspersoft package there?